tests/configurableField.py

```diff
@@ -94,15 +96,15 @@ def testBasics(self):
         c.c1 = Config1
         self.assertEqual(c.c1.f, 5)
 
-        f= Config2(**dict(c.items()))
+        f= Config2(**dict(list(c.items())))
```

Comment:
No. Sometimes the changes come through thick and fast and I don't spot everything. `futurize` plays it absolutely safe and declares that if it returned a list on python2 it will also do it afterwards.